la_palma_overview/__init__.py

```diff
@@ -66,6 +66,19 @@ def clock2img(rows, cols):
         transform=ax.transAxes,
     )
 
+    try:
+        run_id = sfc.main_page().run_id
+        if run_id is not None:
+            ax.text(
+                0.5, 0.0, 'Run {0: 3d}'.format(run_id),
+                horizontalalignment='center',
+                verticalalignment='center',
+                fontsize=60, color='red',
+                transform=ax.transAxes,
+            )
+    except:
+        log.exception('exception in clock2img:')
```

Comment:
Right, I was puzzled by the fact, that I wasn't able to call it without any argument. ... I think empty string is enough in principle...